### DISCUSSION TITLE
Unable to send mail using TLS encription

### DISCUSSION THREAD

**Discussion by vishalrijhwani** [cachethq/cachet#4387] [Q&A]
Hello,
         Good morning, i am facing issue sending mail with tls 
My mail configuration in environment variable is as follows
Mail_PORT: 10025
Mail DRIVER: smtp
Mail_HOST: xxxxx.xxxxxx.xxxxx
Mail_USERNAME: xxxxxxxxxxx (just username not in format XXXXX@XXXXXXXX)
Mail_PASSWORD: xxxxxxxxxxx 
Mail_Address:  xxxxxxxx@xxxxxxxxxxx
Mail_Name: same as mail address
Mail_Encrption: tls

I have put my certificate in /etc/ssl/certs folder too but when i try to send mail i get following error
[object] (ErrorException(code: 0): stream_socket_enable_crypto(): SSL operation failed with code 1. OpenSSL Error messages:
error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed at /var/www/html/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Transport/StreamBuffer.php:94)

Any help will be appreciated.

**Comment by vishalrijhwani** [comment]
I am using 2.4 version of cachetq

**Comment by jbrooksuk** [comment]
@vishalrijhwani this looks to be an issue with your mail provider or your server not being able to verify the certificate.

What provider are you using?

**Comment by vishalrijhwani** [comment]
My email provider is my company mail i.e fisglobal.com. you are right certificate verification failed. Please suggest or advise any possible solution
